model/ddl.go

```diff
@@ -215,8 +215,11 @@ type Job struct {
 	SchemaState SchemaState     `json:"schema_state"`
 	// SnapshotVer means snapshot version for this job.
 	SnapshotVer uint64 `json:"snapshot_ver"`
-	// StartTS uses timestamp allocated by TSO.
+	// CreateTS uses timestamp allocated by TSO.
 	// Now it's the TS when we put the job to TiKV queue.
+	CreateTS uint64 `json:"create_ts"`
```

Comment:
I think we'd better don't change the `StartTS` in the `Job` structure that may cause incompatibility between the old and new versions. We can change it when executing showing DDL jobs.

Reply:
this means we needn't add the variable`CreateTS`?